src/server.cpp

```diff
@@ -1673,6 +1673,10 @@ void Server::SendHUDChange(session_t peer_id, u32 id, HudElementStat stat, void
 		case HUD_STAT_SIZE:
 			pkt << *(v2s32 *) value;
 			break;
+		case HUD_STAT_BOLD:
+		case HUD_STAT_ITALIC:
+			pkt << *(bool *) value;
```

Comment:
`pkt << *(bool *)value;` -- no space between cast and value.

Reply:
Sorry, I don't understand what's wrong here. Should there be no space between cast and value?

Reply:
Yeah, that's all.  It's not in https://dev.minetest.net/Code_style_guidelines (probably because no one has added it yet, and I don't have a dev.minetest.net wiki account yet) or the Linux kernel guidelines, but it's still a rule we have.

Reply:
It's up to you.  Honestly, since there's no rule about it, you could probably leave it as-is.

Reply:
Ok, since I've seen casts with a space at other places in the code, I'm going to leave it like that.